### PR TITLE
Changed PIP003 in according to new basic income claim process.

### PR DESCRIPTION
**Pull request by mateleshkavo**

### CODE REVIEW

File: PIPS/pip-003.md [hunk 1]
```diff
@@ -9,24 +9,42 @@
...
 
-To claim the income, members can submit a claim through [the reward store](https://www.heytaco.chat/rewards).
+To claim the income, members can submit a claim through [the TITs Claim Sheet](https://docs.google.com/forms/d/e/1FAIpQLScPyotEu_tvSH5DY3tD6Q130OixWRDrStrX_YoKG4viDApE9w/viewform).
 
 Claims can be made with following taco amounts:
-- 4 🌮 give 30,000 PSC
-- 8 🌮 give 60,000 PSC
-- 12 🌮 give 90,000 PSC
-- 16 🌮 give 120,000 PSC
+- 4 🌮 give 4 NFT
```
Comment (sunify) [inline PIPS/pip-003.md]: NFT? Maybe TIT?

**Comment by troggy** [comment]
this is outdated again

**Comment by johannbarbie** [comment]
to make it simple, i would not write anything about PSC or DAI, only say that we tokenize each weeks taco's using TITs, and that TITs can be used in different reward schemes, as long as these reward schemes exists. how about that?

**Comment by sunify** [comment]
I would avoid `TITs` writing (`TIT` much better). It's kinda sexist and not so welcoming. Which is bad for an open community that we want to build.

It's a fun joke, but probably we shouldn't put such things in our formal docs.

Btw, we do not write `PSCs`, `ETHs` and so on.

**Pull request merged**